pretest/edc-openapi.js

```diff
@@ -1,21 +1,30 @@
-const fs = require('fs/promises');
+const fs = require(`fs/promises`);
 
 const folder = "node_modules";
 
-const context = "management-api";
-const resourceUrl = `https://eclipse-edc.github.io/Connector/openapi/${context}/${context}.yaml`;
+const CONTEXTS = [
+  {
+    name: "management-api",
+    resourceUrl: "https://eclipse-edc.github.io/Connector/openapi/management-api/management-api.yaml"
+  },
+  {
+    name: "identity-hub-api",
+    resourceUrl: "https://eclipse-edc.github.io/IdentityHub/openapi/identity-api/identity-api.yaml"
+  }
+]
 
-fetch(resourceUrl)
-  .then(response => {
-    if (!response.ok) {
-      throw new Error(`Failed to download ${context} openapi spec`);
-    }
-    return response.text()
-  })
-  .then(text => text
-    .replaceAll("example: null", "")
-    .replaceAll("https://w3id.org/edc/v0.0.1/ns/value", "value")) // to make "secrets" test working, will be fixed in the future
-  .then(text => fs.writeFile(`${folder}/${context}.yml`, text))
-  .catch(error => {
-    console.error('Error downloading openapi specs:', error);
-  })
+Promise.all(CONTEXTS.map((context) =>
+  fetch(context.resourceUrl)
+    .then(response => {
+      if (!response.ok) {
+        throw new Error(`Failed to download ${context.name} openapi spec`);
+      }
+      return response.text()
+    })
+    .then(text => text
+      .replaceAll("example: null", "")
+      .replaceAll("https://w3id.org/edc/v0.0.1/ns/value", "value")) // to make "secrets" test working, will be fixed in the future
```

Comment:
these two lines can be removed as a fix was provided in the EDC openapi generator, we could do it in a separate pr, but it is a really simple fix, if you agree let's amend it here